packages/subgraph/schema.graphql

```diff
@@ -1656,6 +1688,43 @@ type StreamPeriod @entity {
     totalAmountStreamed: BigInt
 }
 
+"""
+DefaultGovernanceConfigs: A higher order entity that represents the default governance configs.
+When a config is set in governance with superToken field as address(0), it will be used as the default config.
+"""
+type DefaultGovernanceConfigs @entity {
```

Comment:
So there is this one default configuration entity? And when there's anything token specific then that is saved on the token?

Reply:
yes